fundamental-concepts/propose-accept-pattern/5_demo.md

```diff
@@ -0,0 +1,9 @@
+You can now [open the UI
+tab](https://[[HOST_SUBDOMAIN]]-3000-[[KATACODA_HOST]].environments.katacoda.com).  You are
+presented with the `create-daml-app` login screen. The following screencast shows off the new
```

Comment:
```suggestion
You can now [open the UI
tab](https://[[HOST_SUBDOMAIN]]-3000-[[KATACODA_HOST]].environments.katacoda.com).  Note that it might take a few moments before the UI shows up. You are
presented with the `create-daml-app` login screen. The following screencast shows off the new
```

Reply:
added.